.git-blame-ignore-revs

```diff
@@ -0,0 +1 @@
+01addd4d124b739eb4a560f841cd4e798aa1e36d
```

Comment:
for ignoring reformatting commits in `git blame` - is this file name the default, or would i need to specify it as CLI argument (or in the config)?

Reply:
You have nothing to do neither locally nor on github. This change is also existing on all other modules which were migrated to use black.